pebblo/app/service/discovery_service.py

```diff
@@ -145,11 +151,34 @@ def process_request(self):
             )
             self._write_file_content_to_path(ai_apps.dict(), file_path)
 
-            logger.debug("AiApp discovery request completed successfully")
-            return {"message": "App Discover Request Processed Successfully"}
+            ai_apps_data = ai_apps.dict()
+            ai_apps_obj = DiscoverAIApps(
```

Comment:
Some of these fields are Optional in the AiApp model, the code will raise a KeyError.

Reply:
Updated